evcouplings/compare/sifts.py

```diff
@@ -137,8 +140,37 @@ def find_homologs_jackhmmer(**kwargs):
     if config["prefix"] is None:
         config["prefix"] = path.join(tempdir(), "compare")
 
-    # run jackhmmer against sequence database
-    ar = jackhmmer_search(**config)
+    check_required(
+        config,
+        [
+            "database", "prefix", "use_bitscores",
+            "domain_threshold", "sequence_threshold",
+            "nobias", "cpu",
+        ]
+    )
+
+    # run hmmsearch (possibly preceded by hmmbuild)
+    if not use_jackhmmer:
```

Comment:
access using config["use_jackhmmer"]; also just flip the cases, the "not" here is not necessary